tools/bin/setup_zksolc.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+# Specify the version of zksolc you want to install
+VERSION="1.5.6"
```

Comment:
What do you think about parameterising this for both the Zksolc setup and the Solidity compilation script so that the products and the user can choose the version of the contracts to be compiled? for example ccip 1.5.3, automation could be a different version, even within ccip, MCMS contracts have a different version

Reply:
fair

Reply:
done